api/pom.xml

```diff
@@ -2,7 +2,7 @@
    <parent>
       <groupId>org.openmrs</groupId>
       <artifactId>openmrs</artifactId>
-      <version>1.12.0.SPRING-SNAPSHOT</version>
+      <version>1.12.0-SPRING4-SNAPSHOT</version>
```

Comment:
It's needed for testing of the spring-4 branch. It will be back 1.12.0-SNAPSHOT once we merge with master.